test/Tests/src/System/File_Spec.enso

```diff
@@ -43,6 +43,13 @@ spec =
             non_existent_file.exists.should_be_false
             sample_file.exists.should_be_true
 
+        Test.specify "should get file size" <|
+            sample_file.size.should_equal 403
+
+            result = non_existent_file.size
+            result.should_fail_with File_Error
+            result.catch.should_be_a File_Error.Not_Found
```

Comment:
Please add a test for computing the size of directory, as specified in the doc of the function.

It's a non-trivial part of this feature, no way to know for sure if it works if we don't have a test for it.